src/Tools/dotnet-monitor/Program.cs

```diff
@@ -58,12 +58,12 @@ class Program
                 Argument = new Argument<bool>(name: "metrics", getDefaultValue: () => true )
             };
 
-        private static Option ReversedServerAddress() =>
+        private static Option DiagnosticPort() =>
             new Option(
-                alias: "--reversed-server-address",
-                description: "A fully qualified path and filename for the OS transport to communicate over.")
+                alias: "--diagnostic-port",
+                description: "A fully qualified path and filename of the listening diagnostic port to which runtime instances can connect.")
```

Comment:
Reverse Server --> Connect Diagnostic Port if we follow the terminology from the runtime.  I might forgo that phrasing entirely and just say "Diagnostic Port".

Reply:
Can you write out the before and after? I'm not certain what you want me to change.